test/unit/integration.js

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2020 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const assert = require("assert");
+
+const itCallbacks = [];
+function it(desc, callback) {
+  itCallbacks.push([desc, callback]);
+}
+
+const testsCallback = [];
+function describe(desc, filename, id, callback) {
+  testsCallback.push([desc, filename, id, callback]);
+}
+
+function selector(id) {
+  return `#\\3${id.charAt(0)} ${id.slice(1)}`;
+}
+
+async function runMe(baseUrl, browser) {
+  // Create the pages
+  await Promise.all(
+    testsCallback.map(([_, filename, id, cb]) =>
```

Comment:
We don't seem to be using the `id` here at all. Removing that would also make this helper more generally useful for other integration tests.

Reply:
the id is used as a selector to select the element to wait for before running the test.